Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/security-emailthreatsubmission-get.md

---
title: "Get emailThreatSubmission"
description: "Read the properties and relationships of an emailThreatSubmission object."
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

# Get emailThreatSubmission
Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Read the properties and relationships of an [emailThreatSubmission](../resources/security-emailthreatsubmission.md) object.

## Permissions

> One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

Reviewer: ```suggestion
One of the following permissions is required to call this API. To learn more permissions, including how to choose one, see [Permissions](/graph/permissions-reference).
```

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|ThreatSubmission.Read, ThreatSubmission.ReadWrite, ThreatSubmission.Read.All, ThreatSubmission.ReadWrite.All|

> |Delegated (personal Microsoft account)|N/A|

Reviewer 2: Line 23: Instead of "N/A" please use "Not supported."

Please apply to all topics in this PR.

|Application|ThreatSubmission.Read.All, ThreatSubmission.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /security/threatSubmission/emailThreats/{emailThreatsId}
```

## Optional query parameters
Not suppported.

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body

> Do not supply a request body for this method.

Reviewer: ```suggestion
A request body is not required for this method. 
```
